prac_06/guitars.py

```diff
@@ -0,0 +1,33 @@
+"""cp1404 practical for guitair program"""
+
+from prac_06.guitar import Guitar
+
+
+def main():
+    guitars = []
+
+    print("My Guitars!")
+    name = input("Name: ")
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        guitar_to_add = Guitar(name, year, cost)
+        guitars.append(guitar_to_add)
+        print(guitar_to_add, "Added")
+        name = input("Name: ")
+
+    guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
+    guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+
+    if guitars:
```

Comment:
Some of this looks too similar to the solutions; including this part, which was not in the requirements/question.